lib/linguist/languages.yml

```diff
@@ -4013,6 +4013,12 @@ Processing:
   tm_scope: source.processing
   ace_mode: text
   language_id: 294
+Proguard:
+  type: programming
+  color: "#0096D8"
```

Comment:
Maybe we could try one of the orange colors from the logo?

Reply:
I'm not sure that orange logo that shows up in Google is official. The website is pretty much white / blue: https://www.guardsquare.com/en/products/proguard

Reply:
Ah... So many languages want the blue color...
Though, since this is now a data language, we probably don't need a color. It would only show up if someone uses `linguist-detectable` on Proguard anyway.

Reply:
:+1: removed color